server/pipeline_friend.go

```diff
@@ -100,7 +103,39 @@ func (p *pipeline) addFacebookFriends(logger *zap.Logger, userID []byte, accessT
 				if err != nil {
 					logger.Error("Could not commit transaction", zap.Error(err))
 				} else {
-					logger.Info("Imported friends from Facebook")
+					logger.Debug("Imported friends from Facebook")
+
+					// Send out notifications.
+					if len(friendUserIDs) != 0 {
+						content, err := json.Marshal(map[string]interface{}{"handle": handle, "facebook_id": fbid})
```

Comment:
Should this also include the user_id as well? I know they can query the accounts by handle, I'm just playing devil's advocate here